src/pages/resultsView/survival/logRankTest.ts

```diff
@@ -309,6 +309,89 @@ export function logRankTest(...survivalCurves: SurvivalCurveItem[][]) {
             };
         })
         .filter(x => x.d > 0);
-
     return _logRankTest(survivalItems, groupsTte, groupsEv).pValue;
 }
+
+export function calculatePairWiseHazardRatio(
+    controlGroup: string,
+    ...survivalCurves: SurvivalCurveItem[][]
+) {
+    const allEvents = _.sortBy(_.flatten(survivalCurves), s => s.months);
```

Comment:
this is important business logic here so comments about what we're doing would be helpful to devs trying to understand the basics